### DISCUSSION TITLE
How to use bun:sqlite?

### DISCUSSION THREAD

**Discussion by Jeesson** [electron-vite/electron-vite-vue#483] [Q&A]
Faced a problem using bun:sqlite.

some recommendations have been made, such as:
- @types/bun installed
- added types to tsconfig.json `"types": ["bun-types"]` or `"types": ["@types/bun"]`. In both cases types works, autocompletions are available
- run with `bun --bun vite`

```
[plugin:vite:import-analysis] Failed to resolve import "bun:sqlite" from "src/database.ts". Does the file exist?
```
```
node:internal/modules/cjs/loader:1084
Uncaught Error: Cannot find module 'bun:sqlite'
Require stack:
- electron/js2c/renderer_init
...
```

**Comment by caoxiemeihao** [comment]
I couldn't find any information about Vite supporting `bun`. Can you provide a Demo?

**Reply by Jeesson** [reply]
Oh, [it says](https://bun.sh/guides/ecosystem/vite) that although Vite currently works with Bun, it hasn't been heavily optimised, and Vite hasn't been adapted to use Bun's bundler, module resolver or transpiler.
I guess we'll just have to wait, right?
